src/sentry/api/serializers/models/rule.py

```diff
@@ -134,22 +130,11 @@ def get_attrs(self, item_list, user, **kwargs):
             group_by="uuid",
         )
 
-        for item in item_list:
-            if item.owner_id is not None:
-                owners_by_type[actor_type_to_string(item.owner.type)].append(item.owner_id)
-
-        for k, v in ACTOR_TYPES.items():
-            actors = Actor.objects.filter(type=v, id__in=owners_by_type[k])
-            if k == "team":
-                resolved_actors[k] = {actor.id: actor.team_id for actor in actors}
-            if k == "user":
-                resolved_actors[k] = {actor.id: actor.user_id for actor in actors}
-
         for rule in rules.values():
-            if rule.owner_id:
-                type = actor_type_to_string(rule.owner.type)
-                if rule.owner_id in resolved_actors[type]:
-                    result[rule]["owner"] = f"{type}:{resolved_actors[type][rule.owner_id]}"
+            if rule.owner_team_id or rule.owner_user_id:
+                actor = ActorTuple.from_id(user_id=rule.owner_user_id, team_id=rule.owner_team_id)
+                if actor:
+                    result[rule]["owner"] = actor.identifier
```

Comment:
In crons we serialized this using the actor serializer. Should we do that here too?

Reply:
https://github.com/getsentry/sentry/blob/c601acee1b344a608ea4b491d7cdc5002cc7db5b/src/sentry/monitors/serializers.py#L184-L191

https://github.com/getsentry/sentry/blob/c601acee1b344a608ea4b491d7cdc5002cc7db5b/src/sentry/monitors/serializers.py#L220